src/main/java/com/gmail/inverseconduit/BotConfig.java

```diff
@@ -103,6 +123,10 @@ public Path getJavadocsDir() {
         return javadocs;
     }
 
+    public List<Long> getAdmins() {
```

Comment:
I'm not liking this, we should separate the method into `isAdmin(user)`, `removeAdmin(user)` and `addAdmin(user)` so that we can implement automatically saving the changes (in the latter two parts) without deprecating old code.

If doing it in the BotConfig file isn't the right way I propose:

1. convert `getAdmins` into `default` privacy
2. creating and using `Permissions` class in the same package
3. create the middleman functions in it